style.css

```diff
@@ -0,0 +1,901 @@
+}
+
+.mob-list {
+  height: 50%;
+  display: flex;
+  flex-direction: column;
+  margin-top: 1rem;
+  margin-left: 1.5rem;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.mob-list li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: start;
+  width: 100%;
+  font-size: 24px;
+  font-weight: 500;
+  padding-top: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+.portfolio-link {
+  cursor: pointer;
+  color: #272a31;
+}
+
+.portfolio-link-a {
+  text-decoration: none;
+  width: 100%;
+  height: 3rem;
+  padding-right: 6rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #ddd;
+}
+
+.portfolio-link-a:visited {
+  color: #d3d3d3;
+}
+
+.portfolio-link-a:hover {
+  color: #d3d3d3;
+}
+
+.new-link {
+  transition: all 0.5s 0s ease;
+}
+
+.new-link:hover {
+  padding-left: 20px;
+}
+
+/* ========== mobile-veiw =============== */
+
+/* =========== first section About page ======== */
+
+.about-section {
+  background-image: url(img/mobile_intro_bg.png);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: auto;
+  margin-bottom: 5rem;
+}
+
+.h1,
+.h2 {
+  width: 100%;
+  text-align: center;
+  padding: 1rem;
+  margin: 1rem auto;
+}
+
+.article-1-about {
+  color: #272a31;
+  margin: 1rem auto;
+  width: 85%;
+  height: 35vh;
+  text-align: center;
+  padding: 1rem;
+  background-color: white;
+}
+
+.date-about,
+.place-about {
+  font-size: medium;
+  width: 100%;
+  text-align: center;
+  margin-top: 3rem;
+}
+
+.place-about {
+  border-bottom: 1px solid #272a31;
+  width: fit-content;
+  margin: 0 auto;
+}
+
+.h1-global {
+  text-align: center;
+}
+
+.hr-about-2 {
+  width: 5%;
+  margin: 0 auto;
+  color: #ec5342;
+}
+
+.hr-about-15 {
+  display: none;
+}
+
+.h1-about {
+  width: 100%;
+  margin-top: 2rem;
+  font-size: large;
+}
+
+.hr-about {
+  color: #ec5342;
+  width: 5%;
+  margin: 0.5rem auto;
+}
+
+.articl-2 {
+  width: 85%;
+  margin: 2rem auto;
+  color: #383a3f;
+  text-align: center;
+}
+
+.art-logo {
+  display: block;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.hr-about-4 {
+  color: #ec5342;
+  width: 5%;
+  margin: 0 auto;
+}
+
+.cop26-1 {
+  display: block;
+  width: 60%;
+  margin: 1rem auto;
+  filter: grayscale(100%) brightness(40%) sepia(100%) hue-rotate(-50deg) saturate(600%) contrast(0.8);
+  transition: 200ms;
+}
+
+.cop26-1:hover {
+  filter: none;
+  transform: translateX(5px);
+}
+
+.img-container {
+  width: 100%;
+  position: relative;
+  text-align: center;
+  color: white;
+  cursor: pointer;
+  font-size: large;
+  font-weight: 500;
+}
+
+.center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.footer-h {
+  width: 50%;
+  margin: 1rem auto;
+  text-align: center;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: #3c3f46;
```

Comment:
- [ ] Kindly follow the given design.
## Your Design:
![image](https://user-images.githubusercontent.com/92215878/192033395-9bda1176-b03b-4cf6-97ae-9af649e71caf.png)

## Given Design:
![image](https://user-images.githubusercontent.com/92215878/192033225-26c6370a-d316-4048-abef-065a3925beda.png)